dispatcher/MongoConnect.py

```diff
@@ -304,10 +304,11 @@ def is_timeout(self, doc, t):
             self.log.debug(f'{host} last reported {int(dt)} sec ago')
             ret = ret or True
         if has_ackd is not None and t - has_ackd > self.timeout_take_action:
-            self.log.debug(f'{host} hasn\'t ackd a command from {int(t-has_ackd)} sec ago')
+            self.log.info(f'{host} hasn\'t ackd a command from {int(t-has_ackd)} sec ago')
             if self.host_config[host] == 'tpc':
                 dt = (now() - self.hv_timeout_fix[host]).total_seconds()
                 if dt > self.hypervisor_host_restart_timeout:
+                    self.log_error(f'Hypervisor fixes timeout of {host}', "ERROR", "ERROR")
```

Comment:
This `log_error` function needs redoing at some point because it isn't used consistently

Reply:
Please feel free to edit the PR if you think this is the opportune moment to implement this